S/SoftHSM2/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SoftHSM2"
+version = v"2.6.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://dist.opendnssec.org/source/softhsm-$(version).tar.gz", "61249473054bcd1811519ef9a989a880a7bdcc36d317c9c25457fc614df475f2")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd softhsm-2.6.1/
+export LDFLAGS="-L${libdir}"
+export CPPFLAGS="-I${includedir}"
```

Comment:
I don't think we need to set these flags nowadays
```suggestion
cd $WORKSPACE/srcdir/softhsm-*
```